src/gui/src/displayControls.cpp

```diff
@@ -293,41 +293,62 @@ DisplayControls::DisplayControls(QWidget* parent)
+  // stdcell instances
+  auto stdcell_parent = makeParentItem(
+      instances_.stdcells,
+      "StdCells",
+      instance_group_.name,
       Qt::Checked,
       true);
 
-  // make instance items, non-null last argument to create checkbox
-  makeLeafItem(functional_instances_.combinational, "Combinational", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.sequential, "Sequential", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.buffer_inv, "Buffers/Inverters", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.clock_gate, "Clock gates", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.levelshifter, "Level shifters", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.pad, "Pad", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.macro, "Macro", func_instances_parent, Qt::Checked, true);
-  makeLeafItem(functional_instances_.memory, "Memory", func_instances_parent, Qt::Checked, true);
-  toggleParent(functional_instance_group_);
+  auto bufinv_parent = makeParentItem(
+      stdcell_instances_.bufinv,
+      "Buffers/Inverters",
+      stdcell_parent,
+      Qt::Checked,
+      true);
+  makeLeafItem(bufinv_instances_.timing, "Timing opt.", bufinv_parent, Qt::Checked, true);
+  makeLeafItem(bufinv_instances_.other, "Netlist", bufinv_parent, Qt::Checked, true);
+  toggleParent(stdcell_instances_.bufinv);
+
+  makeLeafItem(stdcell_instances_.combinational, "Combinational", stdcell_parent, Qt::Checked, true);
+  makeLeafItem(stdcell_instances_.sequential, "Sequential", stdcell_parent, Qt::Checked, true);
+  auto clock_tree_parent = makeParentItem(
+      stdcell_instances_.clock_tree,
+      "Clock tree",
+      stdcell_parent,
+      Qt::Checked,
+      true);
+  makeLeafItem(clock_tree_instances_.bufinv, "Buffers/Inverters", clock_tree_parent, Qt::Checked, true);
+  makeLeafItem(clock_tree_instances_.clock_gates, "Clock gates", clock_tree_parent, Qt::Checked, true);
+  toggleParent(stdcell_instances_.clock_tree);
+
+  makeLeafItem(stdcell_instances_.level_shiters, "Level shifters", stdcell_parent, Qt::Checked, true);
+
+  auto phys_parent = makeParentItem(
+      stdcell_instances_.physical,
+      "Physical",
+      stdcell_parent,
+      Qt::Checked,
+      true);
+  makeLeafItem(physical_instances_.fill, "Fill cells", phys_parent, Qt::Checked, true);
+  makeLeafItem(physical_instances_.endcap, "Endcaps", phys_parent, Qt::Checked, true);
+  makeLeafItem(physical_instances_.tap, "Welltaps", phys_parent, Qt::Checked, true);
+  toggleParent(stdcell_instances_.physical);
+  toggleParent(instances_.stdcells);
+
+  makeLeafItem(instances_.blocks, "Macros", instances_parent, Qt::Checked, true);
+  makeLeafItem(instances_.pads, "Pads", instances_parent, Qt::Checked, true);
+  makeLeafItem(instances_.cover, "Cover", instances_parent, Qt::Checked, true);
```

Comment:
Should the physical instance be moved from under the standard cells to instances. Since cover usually isn't part of the standard cells?

Reply:
I think so.

Reply:
Personally I'm willing to overlook this one. I feel that cover cells are essentially a type of standard cell, just not a core logic cell.

Reply:
@rovinski  I don't see it that way.  I would either leave it as is or move physical up a level (I prefer the latter).

Reply:
It's not perfect either way because then the other physical cells aren't under standard cells 🤷‍♂️

Reply:
I moved the up one level per @maliberty preference. I think it would be difficult to perfectly split out the physical cells into standard cells and non-standard cells and maintain the ease of just pressing once to turn them off.